geomesa-kafka/geomesa-kafka-datastore/geomesa-kafka-08-datastore/pom.xml

```diff
@@ -119,12 +126,13 @@
         <dependency>
             <groupId>org.apache.curator</groupId>
             <artifactId>curator-test</artifactId>
-            <version>2.7.1</version>
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>joda-time</groupId>
-            <artifactId>joda-time</artifactId>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-kafka-datastore-common_2.11</artifactId>
```

Comment:
It was my (possibly mistaken) understanding that @tkunicki 's script required that internal geomesa dependencies should have a literal ``_2.11`` and that ``${scala.binary.version}`` should be used with external dependencies like Kafka, Specs2, etc. See for example the Accumulo DS pom. If I am wrong, please correct!